man/systemd-resolved.service.xml

```diff
@@ -104,39 +104,55 @@
+      </para></listitem>
 
       <listitem><para>Multi-label names with the domain suffix <literal>.local</literal> are routed to all
-      local interfaces capable of IP multicasting, using the MulticastDNS protocol. As with LLMNR IPv4
-      address lookups are sent via IPv4 and IPv6 address lookups are sent via IPv6.</para></listitem>
+      local interfaces capable of IP multicasting, where MulticastDNS is not disabled, using the MulticastDNS
+      protocol. As with LLMNR, IPv4 address lookups are sent via IPv4 and IPv6 address lookups are sent via
+      IPv6.</para></listitem>
+
+      <listitem><para>Resolution of address records (A and AAAA) for non-synthesized single-label names is
+      only allowed for non-top-level domains. This means that such records can only be resolved when search
+      domains are defined. For any interface which defines search domains, such look-ups are routed to that
+      interface, suffixed with each of the search domains defined on that interface in turn. When global
+      search domains are defined, such look-ups are routed to all interfaces, suffixed by each of the global
+      search domains in turn. The details of which servers are queried and how the final reply is chosen are
+      described below. Note that this means that address queries for single-label names are never sent out to
+      remote DNS servers, and if no search domains are defined, resolution will fail.</para></listitem>
 
       <listitem><para>Other multi-label names are routed to all local interfaces that have a DNS server
-      configured, plus the globally configured DNS server if there is one. Address lookups from the
-      link-local address range are never routed to DNS. Note that by default lookups for domains with the
-      <literal>.local</literal> suffix are not routed to DNS servers, unless the domain is specified
-      explicitly as routing or search domain for the DNS server and interface. This means that on networks
-      where the <literal>.local</literal> domain is defined in a site-specific DNS server, explicit search or
-      routing domains need to be configured to make lookups within this DNS domain work. Note that today it's
-      generally recommended to avoid defining <literal>.local</literal> in a DNS server, as <ulink
+      configured, plus the globally configured DNS servers if there are any. Note that by default lookups for
+      domains with the <literal>.local</literal> suffix are not routed to DNS servers, unless the domain is
+      specified explicitly as routing or search domain for the DNS server and interface. This means that on
+      networks where the <literal>.local</literal> domain is defined in a site-specific DNS server, explicit
+      search or routing domains need to be configured to make lookups within this DNS domain work. Note that
+      today it's generally recommended to avoid defining <literal>.local</literal> in a DNS server, as <ulink
```

Comment:
```suggestion
      today it's generally recommended to avoid defining <literal>.local</literal> in a DNS server, as <ulink
```
```suggestion
      these days, it's generally recommended to avoid defining <literal>.local</literal> in a DNS server, as <ulink
```